Dockerfile

```diff
@@ -1,28 +1,18 @@
-FROM ubuntu:14.04
-MAINTAINER Vaibhav Bhembre <vaibhav@digitalocean.com>
-
-ENV GOROOT /goroot
-ENV GOPATH /go
-ENV PATH $GOROOT/bin:$PATH
-ENV APPLOC $GOPATH/src/github.com/digitalocean/ceph_exporter
-
-RUN apt-get install -y apt-transport-https
-
-RUN echo "deb https://download.ceph.com/debian-jewel trusty main" >> /etc/apt/sources.list
+FROM ubuntu:16.04
```

Comment:
Most of the builds I am working with are still using `14.04`. Is there any specific advantage you are looking at for moving ahead or just trying to keep things at their latest? I would prefer to move this in time otherwise, consistently with rest of our builds.